script/release-tag-git

```diff
@@ -104,7 +104,7 @@ fi
 
 # Push the tag to the server.
 set -x
-git push https://github.com/cloudendpoints/esp ${VERSION_TAG}
+git push upstream ${VERSION_TAG}
```

Comment:
do we have problem using "https://github.com/cloudendpoints/esp" directly?

this script doesn't require you need to setup "upstream" remote.

Reply:
yes, https://github.com/cloudendpoints/esp doesn't work for me. Did it work for you?

git@github.com:cloudendpoints/esp.git will work for me. And we use "upstream" in a couple of other scripts

Reply:
I guess it is OK to use upstream.